sdks/python/apache_beam/runners/portability/fn_api_runner.py

```diff
@@ -1173,16 +1173,20 @@ class FnApiMetrics(metrics.metric.MetricResults):
   def __init__(self, step_metrics):
     self._counters = {}
     self._distributions = {}
-    for step_metric in step_metrics.values():
-      for proto in step_metric.user:
-        key = metrics.execution.MetricKey.from_runner_api(proto.key)
-        if proto.HasField('counter_data'):
-          self._counters[key] = proto.counter_data.value
-        elif proto.HasField('distribution_data'):
-          self._distributions[
-              key] = metrics.cells.DistributionResult(
-                  metrics.cells.DistributionData.from_runner_api(
-                      proto.distribution_data))
+    for step, step_metric in step_metrics.items():
+      print step
```

Comment:
<!--thread_id:cc_164922562_t; commit:ea8ec4f5bdb63253febaa6bcebf20e839c39749d; resolved:1-->
<!--section:context-quote-->
> **robertwb** wrote:
> debugging

<!--section:body-->
Uff. Sorry about that. Removed.